HUD/PauseMenuGetUniqueIdOfMouseHoveredSlot.md

```diff
@@ -0,0 +1,15 @@
+---
+ns: HUD
+aliases: ["0x13C4B962653A5280"]
+---
+## _PAUSE_MENU_GET_UNIQUEID_OF_MOUSE_HOVERED_SLOT
```

Comment:
This name does not match your filename. There should be another `_` between `UNIQUE` and `ID` (`_PAUSE_MENU_GET_UNIQUE_ID_OF_MOUSE_HOVERED_SLOT`).